src/screens/BackupSheet.js

```diff
@@ -133,20 +112,10 @@ const BackupSheet = () => {
     onSuccess,
   ]);
 
-  const onManualBackup = useCallback(() => {
-    switchSheetContentTransitionRef.current?.animateNextTransition();
-    setStep(WalletBackupStepTypes.manual);
-    setOptions({
```

Comment:
We're no longer using this logic, correct?

Reply:
correct! i refactored how we handle sheet's height for each step.